src/ray/core_worker/core_worker.cc

```diff
@@ -524,7 +533,7 @@ Status CoreWorker::SubmitTask(const RayFunction &function,
   BuildCommonTaskSpec(
       builder, worker_context_.GetCurrentJobID(), task_id,
       worker_context_.GetCurrentTaskID(), next_task_index, GetCallerId(), function, args,
-      task_options.num_returns, task_options.resources, {},
+      task_options.num_returns, task_options.resources, std::unordered_map<std::string, double>{},
```

Comment:
Is this needed?

Reply:
Reverted.

Reply:
This is actually required on my linux dev machine in order to compile, but seems not necessary on ci machines, maybe because of different versions of compilers .etc.